### ISSUE TITLE
Unable to build release build on Ubuntu 18.04 Bionic

### ISSUE THREAD

**Issue by nelhage** [sorbet/sorbet#2515]
I'm trying to build a release build from Git (at revision 2a16bc6b644e6fc8a309f0b0f31fd3592a02bf1a) on Ubuntu 18.04 Bionic.

I'm observing:
```
[nelhage@hw3:/data/home/nelhage/code/sorbet]$ ./bazel build --config=release-linux //...                                                    [88/552]
WARNING: The following configs were expanded more than once: [static-libs, lto, backtracesymbols, versioned, lto-linux, release-common]. For repeatable flags, repeats are counted twice and may lead to unexpected behavior.
[snip intermediate output]
INFO: Analyzed 2559 targets (1 packages loaded, 562 targets configured).
INFO: Found 2559 targets...
ERROR: /data/home/nelhage/code/sorbet/test/fuzz/BUILD:11:1: Linking of rule '//test/fuzz:fuzz_dash_e_impl' failed (Exit 1)
ld.lld: error: relocation R_X86_64_PC32 cannot be used against symbol absl::container_internal::EmptyGroup()::empty_group; recompile with -fPIC
>>> defined in lto.tmp
>>> referenced by raw_hash_set.h:1773 (external/com_google_absl/absl/container/internal/raw_hash_set.h:1773)
>>>               lto.tmp:(sorbet::realmain::options::Options::Options())

ld.lld: error: relocation R_X86_64_PC32 cannot be used against symbol vtable for sorbet::OSFileSystem; recompile with -fPIC
>>> defined in bazel-out/k8-opt/bin/test/fuzz/_objs/fuzz_dash_e_impl/fuzz_dash_e.pic.o
>>> referenced by FileSystem.h:44 (./common/FileSystem.h:44)
>>>               lto.tmp:(sorbet::realmain::options::Options::Options())

ld.lld: error: relocation R_X86_64_PC32 cannot be used against symbol typeErrorsConsole; recompile with -fPIC
>>> defined in lto.tmp
>>> referenced by memory:3805 (external/llvm_toolchain/bin/../include/c++/v1/memory:3805)
>>>               lto.tmp:(buildInitialGlobalState())

ld.lld: error: relocation R_X86_64_PC32 cannot be used against symbol typeErrorsConsole; recompile with -fPIC
>>> defined in lto.tmp
>>> referenced by memory:3803 (external/llvm_toolchain/bin/../include/c++/v1/memory:3803)
>>>               lto.tmp:(buildInitialGlobalState())
[many more]
```

**Comment by jez** [comment]
I'm having this problem too. I think this is a problem building the version of ruby that we have in sorbet. I think that if you change what you're building to `//main:sorbet` instead of `//...` you can circumvent building ruby and get a release build of just the sorbet binary.

Looking into why it doesn't work right now.

**Comment by nelhage** [comment]
Taking the suggestion from the error message works, for what it's worth:

```
./bazel build --cxxopt=-fPIC --copt=-fPIC --config=release-linux //...
```

but I'm not sure if building everything as PIC has negative performance implications.

**Comment by jez** [comment]
I think this is fixed, but I'm not sure when it was fixed (at least, I regularly build on 20.04 without issue these days). Feel free to reopen if I'm mistaken.
